rest/admin_api.go

```diff
@@ -394,6 +394,52 @@ func (h *handler) handleSetLogging() error {
 	return nil
 }
 
+func (h *handler) handleSGCollectStatus() error {
```

Comment:
I think we should consider returning a JSON message as the status response, with something like
`{"state":"running"}`.  This would simplify backwards compatibility for future enhancements providing additional status information.

Reply:
@adamcfraser you beat me to it.  +1

Reply:
Agreed. I'll update.